components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -94,9 +210,10 @@ class WidgetConfigurationPanel extends React.Component {
             dashboardLayout.selectedItem.config.content[0].props.configs.pubsub.types.indexOf("subscriber") !== -1) {
             for (let [key, val] of pubsubComponent.getPublishersMap().entries()) {
                 this.state.checked = !!(dashboardLayout.selectedItem.config.content[0].props.configs.pubsub.publishers &&
-                dashboardLayout.selectedItem.config.content[0].props.configs.pubsub.publishers.indexOf(val) !== -1);
+                    dashboardLayout.selectedItem.config.content[0].props.configs.pubsub.publishers.indexOf(val) !== -1);
                 publishers.push(<Checkbox id={val}
```

Comment:
Reformat it as 
```
publishers.push(
    <Checkbox
          id={val}
          label={key ...
          ...
    />);
```

Reply:
fixed in 87b573730d4c8021f11a7fc89783b583eed6edbf

Reply:
Please push fixed commits to your branch, so that we can see your fix here.